core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -147,7 +147,9 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
   }
 
   val messagesInRate = newMeter(BrokerTopicStats.MessagesInPerSec, "messages", TimeUnit.SECONDS, tags)
+  val messagesInTotal = newCounter(BrokerTopicStats.MessagesInTotal, tags)
```

Comment:
two questions:
1) will the counter overflow, say if a broker stays alive without rolling bounce for 3 months. 
2) if we bounce a broker, the count will start from zero, how can we make use of those discontinued data points.